resources/public/api/conf/1.0/docs/overview.md

```diff
@@ -1 +1 @@
-**Add overview documentation here**
\ No newline at end of file
+This endpoint allows the developer to submit a declaration that the submission data for a business is complete. A National Insurance number is required.
```

Comment:
This needs to be the overview of the API rather than just the single endpoint we have built. Might need to speak to Mashuk here.

Reply:
Updated with the overview Mashuk has suggested